src/main.rs

```diff
@@ -55,14 +55,7 @@ mod userspace_linker_definitions {
 #[cfg(feature = "userspace")]
 use userspace_linker_definitions::*;
 
-pub(crate) extern "C" fn main(hart_id: usize, device_tree_blob_addr: usize) -> ! {
-    // For now we simply park all the harts other than the boot one
-    if hart_id != Plat::get_primary_hart() {
-        loop {
-            Arch::wfi();
-            core::hint::spin_loop();
-        }
-    }
```

Comment:
We might now want to delete this in this PR, as the rest of the code is not ready for multi-core I assume.

I'll handle it before merging :)